pkg/configuration/configuration.go

```diff
@@ -67,10 +84,45 @@ func initConfig() *Config {
 	return &c
 }
 
-func LoadConfig() *Config {
+func LoadConfig(cl client.Client) *Config {
+	loadFromSecret(cl)
 	return initConfig()
 }
 
+// loadFromSecret retrieves the host operator secret
+func loadFromSecret(cl client.Client) error {
+	// get the secret name
+	secretName := os.Getenv("HOST_OPERATOR_SECRET_NAME")
+	if secretName == "" {
+		return nil
```

Comment:
We could log a warning here. Could help for debugging in case the secret name is not set in prod for some reason.

Reply:
added as part of: d1250e5db929e62d1956927d47ae30ef533a9235